admin_manual/configuration_server/email_configuration.rst

```diff
@@ -33,6 +33,14 @@ Sendmail when your mail server is on the same machine as Nextcloud.
 
 .. _email-smtp-config:
 
+Mail Providers
+--------------
+A mail provider is an app that provides outbound mail service to Nextcloud and allows the sending of system emails
```

Comment:
nit: utilize https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded to show that this feature is new

Reply:
Done